website/docs/r/container_app.html.markdown

```diff
@@ -0,0 +1,337 @@
+
+---
+
+A `header` block supports the following:
+
+* `name` - (Required) The HTTP Header Name.
+
+* `value` - (Required) The HTTP Header value.
+
+---
+
+A `volume_mounts` block supports the following:
+
+* `name` - (Required) The name of the Volume to be mounted in the container.
+
+* `path` - (Required) The path in the container at which to mount this volume.
+
+---
+
+An `ingress` block supports the following:
+
+* `allow_insecure_connections` - (Optional) Should this ingress allow insecure connections?
+
+* `custom_domain` -  // TODO - Add missing `Description` to schema for this property and regenerate this file.
+
+* `fqdn` -  The FQDN of the ingress.
+
+* `is_external` - (Optional) Is this an external Ingress.
+
+* `target_port` - (Required) The target port on the container for the Ingress traffic.
+
+* `traffic_weight` - (Required) A `traffic_weight` block as detailed below.
+
+* `transport` - (Optional) The transport method for the Ingress. Possible values include `auto`, `http`, and `http2`. Defaults to `auto`
+
+---
+
+A `traffic_weight` block supports the following:
+
+~> **Note:** This block only applies when `revision_mode` is set to `Multiple`.
+
+* `label` - (Optional) The label to apply to the revision as a name prefix for routing traffic.
+
+* `latest_revision` - (Optional) This traffic Weight relates to the latest stable Container Revision.
+
+* `revision_suffix` - (Optional) The suffix string to which this `traffic_weight` applies.
+
+* `weight` - (Required) The weight (%) of traffic to send to this revision. 
+
+~> **Note:** The cumulative values for `weight` must equal 100 exactly and explicitly, no default weights are assumed. 
+
+---
+
+A `dapr` block supports the following:
+
+* `app_id` - (Required) The Dapr Application Identifier.
+
+* `app_port` - (Required) The port which the application is listening on. This is the same as the `ingress` port.
+
+* `app_protocol` - (Optional) The protocol for the app. Possible values include `http` and `grpc`. Defaults to `http`.
+
+---
+
+A `registry` block supports the following:
+
+* `password_secret_reference` - (Required) The name of the Secret Reference containing the password value for this user on the Container Registry.
+
+* `server` - (Required) The hostname for the Container Registry.
+
+* `username` - (Required) The username to use for this Container Registry.
+
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the Container App
+
+* `custom_domain_verification_id` - The Custom Domain Verification ID for the Container App.
+
+* `latest_revision_fqdn` - The FQDN of the Latest Revision of the Container App.
+
+* `latest_revision_name` - The name of the latest Container Revision.
+
+* `location` - The location this Container App is deployed in. This is the same as the Environment in which it is deployed.
+
+* `outbound_ip_addresses` - An `outbound_ip_addresses` block as detailed above.
```

Comment:
i don't see this block documented?

Reply:
This should just be a list of IP address strings 🙈